### DISCUSSION TITLE
Kit de communication ma cantine

### DISCUSSION THREAD

**Discussion by valeriemerle** [betagouv/ma-cantine#1159] [Idées]
@gauthierandre  Suite à des retours avec des structures qui peuvent nous aider à faire du déploiement (type de préfectures, des porteurs de PAT...), je me dis que nous aurions besoin d'un kit de communication dédié "ma cantine". Ce kit qui pourrait être mis à disposition sur "ma cantine". Il pourrait comprendre :
la plaquette de présentation pdf
un diaporama de présentation (version courte, version longue)
une vidéo
des phrases d’accroche et des images pour réseaux sociaux....

**Comment by hfroot** [comment]
Relié ? https://github.com/betagouv/ma-cantine/discussions/1045

**Reply by gauthierandre** [reply]
Oui, ça peut. @valeriemerle je mets ça sur Gitbook en attendant
